crates/ruff_workspace/src/configuration.rs

```diff
@@ -1553,6 +1561,34 @@ fn warn_about_deprecated_top_level_lint_options(
     );
 }
 
+/// Detect conflicts between I002 (missing-required-import) and ICN001 (unconventional-import-alias)
+fn conflicting_import_settings(
+    isort: &isort::settings::Settings,
+    flake8_import_conventions: &flake8_import_conventions::settings::Settings,
+) -> Result<()> {
+    use std::fmt::Write;
+    let mut err_body = String::new();
+    for required_import in &isort.required_imports {
+        let name = required_import.qualified_name().to_string();
+        if let Some(alias) = flake8_import_conventions.aliases.get(&name) {
+            writeln!(err_body, "    - {name} -> {alias}").unwrap();
```

Comment:
Maybe add backticks? Not sure though

```suggestion
            writeln!(err_body, "    - `{name}` -> `{alias}`").unwrap();
```